packages/core/src/build-utils/build/loader/probeLoaderPlugin.ts

```diff
@@ -0,0 +1,61 @@
+import { Plugin } from '@rsdoctor/types';
+import type { RuleSetRules } from '@rspack/core';
+import { Loader } from '@rsdoctor/utils/common';
+import { Build } from '@/build-utils';
+import { Utils } from '..';
+
+const BuiltinLoaderName = 'builtin:swc-loader';
```

Comment:
is it ready for "builtin:lightningcss-loader" ?

Reply:
ok， i will add this on another pr，i forgot this issue